new announce gfx, added submenu

ecstech-admin opened the pull request:
Updated the announcements image
Edited announcements bullets
Updated nav to include RP's subnav

ronald-bbg reviewed (changes requested):
Hi Jim - page looks great! Some minor edits:

- Figma has the first row img as sharp corners as opposed to having a border-radius
<img width="586" alt="Screenshot 2023-11-30 at 12 05 18 AM" src="https://github.com/blackberggroup/VUHC/assets/137827146/25dd0186-8623-4c58-80cd-9afdfd1ea87c">

- Figma has different items for the following:
1.   Events has 3 bullets
2.  Img is a one-pager as opposed to a tri-fold
3.  CTA is "View and Download FAQ" instead of "View and Download Brochure"

- Looks like the HEX colors on your branch are correct and figma is incorrect for the VISN headers (@will-bbg for vis)

- In the "Program Eligilibility" row - the image doesn't match figma.

- In the "Program Benefits" row - the man illustration looks to be closer to the lower gray brackground section in figma.

- In the "Publications" carousel, the arrows should match the size and padding like we use in Fellowships and DOE. Hex color should be #3559A5 for the arrows.

- Looks like a bit more padding is needed below the text for each box within the "Multimedia" section.

- Looks also a bit more padding is needed between Our Team, 2nd row and 3rd row.

ecstech-admin commented:
Thanks, @ronald-bbg! Sorry for any confusion. I didn't leave the best comments for what VUHC changes I made in that last commit.

Edits were minor (per Alec):

- created new gfx for "Initiative Announcements and Events" based upon new PDF (tri-fold brochure)
- removed second bullet (of three)
- changed the button text from "View and Download FAQ" to "View and Download Brochure"

The "Program Eligibility" photo was changed in a previous commit

I adjusted the carousel buttons to look more like DoE (although, I left the hex as is since it matches the hex used in "Multimedia" below) but per @will-bbg, I did not touch the other errors you noticed.

ronald-bbg commented:
Confirmed with @ecstech-admin this can be deleted as newer changes already merged to main
